templates/CRM/common/l10n.js.tpl

```diff
@@ -91,10 +91,9 @@
   {literal}
 
   var params = {
-    errorClass: 'crm-inline-error',
+    errorClass: 'error alert-danger',
```

Comment:
I don't see the class `alert-danger` in `civicrm.css`. Where is it defined?

Reply:
@colemanw @mattwire  i think its a standard bootstrap class but that being said i think we should be keeping the errorClass as is as themes may have already styled these errors

Reply:
@colemanw No problem with that - I've changed to `crm-inline-error alert-danger`